firehose-source/src/main/java/source/FirehoseProperties.java

```diff
@@ -0,0 +1,118 @@
+/*
+ *  Copyright 2015 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package source;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Vinicius Carvalho
+ */
+@ConfigurationProperties(prefix = "firehose")
```

Comment:
This whole prefix thing is in (somewhat dormant) discussion. A prefix may be required anyway (for other reasons).
You have a point regarding ease of reading, even though when the module runs alongside other modules, we're likely to provide a way to discern them anyway. But (good) point taken.
For now, I'd remove the prefix so that there is uniformity. We'd of course re-add if this is the decision we take later.